app/views/smart_proxies/show.html.erb

```diff
@@ -0,0 +1,74 @@
+<% javascript('proxy_status') %>
+<% title(_('Smart Proxy: %s') % @smart_proxy.to_label) %>
+<%= smart_proxy_title_actions(@smart_proxy, authorizer) %>
+<% service_features = proxy_features_besides_puppet(@smart_proxy) %>
+<div class="row proxy-show" data-url="<%= ping_smart_proxy_path(@smart_proxy) %>">
+  <ul id="proxy-tab" class="nav nav-tabs">
+    <li class="active"><a href="#properties" data-toggle="tab"><%= _('Overview') %></a></li>
+    <% if service_features.any? %>
+      <li><a href="#services" data-toggle="tab"><%= _('Services') %></a></li>
+    <% end %>
+  </ul>
+  <div id="proxy-tab-content" class="proxy-content tab-content">
+    <div class="tab-pane active in" id="properties">
+      <div class="col-md-6">
+        <div class="row">
+          <h3><%= _('Details') %></h3>
+        </div>
+        <div class="row">
+          <div class="col-md-4">
+            <%= _('Communication status') %>
+          </div>
+          <div class="col-md-8">
+            <span class="proxy-show-status">
+              <%= spinner %>
+            </span>
+          </div>
+        </div>
+        <div class="row">
+          <div class="col-md-4">
+            <%= _('URL') %>
+          </div>
+          <div class="col-md-8">
+            <%= @smart_proxy.url %>
+          </div>
+        </div>
+        <div class="row">
+          <div class="col-md-4">
+            <%= _('Version') %>
+          </div>
+          <div class="col-md-8">
+            <span class="proxy-version" data-url="<%= ping_smart_proxy_path(@smart_proxy) %>">
+              <%= spinner %>
+            </span>
+          </div>
+        </div>
+        <div class="row">
+          <div class="col-md-4">
+            <%= _('Features') %>
+          </div>
+          <div class="col-md-8">
+            <%= @smart_proxy.features.to_sentence %> <%= refresh_proxy_icon(@smart_proxy, authorizer) %>
+          </div>
+        </div>
+      </div>
+      <div class="col-md-6">
+        <!--place holder for Capsule sync-->
+      </div>
+    </div>
+    <% if service_features.any? %>
+      <div class="tab-pane" id="services">
+        <div class="col-md-6">
+          <% service_features.each do |feature| %>
+            <% feature_erb = "smart_proxies/plugins/#{feature}" %>
+            <% if lookup_context.template_exists?(feature_erb, [], true) %>
+              <%= render :partial => feature_erb, :locals => {:feature => feature} %>
+            <% else %>
+              <%= render :partial => "smart_proxies/plugins/no_template", :locals => {:feature => feature} unless feature.start_with?('puppet') %>
```

Comment:
This seems redundant if `service_features` is set to `proxy_features_besides_puppet` already.